app/api/gatcg_spoilers/route.ts

```
@@ -1,61 +1,211 @@
+    }
+  }
+
+  return dataContainer;
+}
+
+async function createOcrScheduler() {
+  const scheduler = createScheduler();
+  const workers = await Promise.all(
+    Array.from({ length: NUM_OCR_WORKERS }, () => createWorker('eng')),
+  );
+  workers.forEach((w) => scheduler.addWorker(w));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)